### PR TITLE
Added new login for dapp

### PR DESCRIPTION
**Pull request by lionellbriones**

### CODE REVIEW

File: app/src/config.js [hunk 1]
```diff
@@ -108,27 +109,24 @@ export default {
...
-      typeOfLogin: TWITCH,
-      name: TWITCH,
-      clientId: VUE_APP_TWITCH_CLIENT_ID,
+      typeOfLogin: TWITTER,
+      name: TWITTER,
+      clientId: VUE_APP_TWITTER_CLIENT_ID,
       logoHover: '',
       logoLight: '',
       logoDark: '',
+      hasLightLogo: false,
```
Comment (chaitanyapotti) [inline app/src/config.js:123]: Add this to types on torus-embed

File: app/src/config.js [hunk 2]
```diff
@@ -108,27 +109,24 @@ export default {
...
       showOnModal: true,
+      jwtParameters: {
+        domain: LOGIN_DOMAIN,
+        connection: 'twitter',
+        isVerifierIdCaseSensitive: false,
+      },
```
Comment (chaitanyapotti) [inline app/src/config.js:129]: Use flags for mobile, desktop separately which enable their visibility

File: app/src/containers/Popup/PopupLogin/PopupLogin.vue [hunk 3]
```diff
@@ -159,7 +142,9 @@ import log from 'loglevel'
 import { mapActions, mapGetters, mapState } from 'vuex'
 
 import PasswordlessLogin from '../../../components/helpers/PasswordLessLogin'
-import { GITHUB, GOOGLE_VERIFIER, PASSWORDLESS, TWITTER } from '../../../utils/enums'
+import { DISCORD_VERIFIER, FACEBOOK_VERIFIER, GITHUB, GOOGLE_VERIFIER, PASSWORDLESS, TWITTER, TWITTER_VERIFIER } from '../../../utils/enums'
+
+const DAPP_MAIN_LOGIN = [FACEBOOK_VERIFIER, TWITTER_VERIFIER, DISCORD_VERIFIER]
```
Comment (chaitanyapotti) [inline app/src/containers/Popup/PopupLogin/PopupLogin.vue:147]: Use direct flags on config

File: app/src/config.js (hunk 2 above)
Comment (lionellbriones) [reply app/src/config.js:141]: added showOnDesktop, showOnMobile

**Review by chaitanyapotti — APPROVED** [review]
great work. lgtm

**Pull request merged**